### PR TITLE
[Closes #164] Execute multi-signature proposal

### LINKED ISSUE #164: Execute actions for a single m-sig proposal
**Is your feature request related to a problem? Please describe.**
Members of the Telos Network do not currently have an easy-to-use interface to generate and sign msigs. Creation heavily relies upon cleos command line knowledge and signing can be done through the EOS Authority explorer but is missing key information about the proposals themselves on the front-end.

**Describe the solution you'd like**
Users can select a single m-sig proposal from the proposals list. This will allow them to see the details of the selected proposals.
If the user is logged in and is one of the accounts listed in the m-sig, they'll be able to execute the actions according to their role. 

**Additional context**
Depends on #161

### PR DESCRIPTION
**Pull request by karynemayer**
# Pull Request Template

## Description

- Makes the authorization field mandatory in the m-sig form
- Includes the page where you can see the details of a single multi-signature proposal
- Allows users to execute, cancel and approve the proposal

Fixes #164 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- [ ] At least one proposal was successfully executed
- [ ] At least one proposal was successfully approved
- [ ] At least one proposal was successfully cancelled
- [ ] New proposal was created
- [ ] Mobile tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/pages/ProposalItem.vue [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+            proposal_name: proposalName,
+            level: {
+              actor: account.value,
+              permission: 'active'
+            }
+          }
+        });
+
+        document.location.reload();
+      } catch (e) {
```
Comment (DonaldPeat) [inline src/pages/ProposalItem.vue:221]: \**

File: src/pages/ProposalItem.vue [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+          name: 'exec',
+          data: {
+            proposer: proposer.value,
+            proposal_name: proposalName,
+            executer: account.value
+          }
+        });
+
+        document.location.reload();
+      } catch (e) {
```
Comment (DonaldPeat) [inline src/pages/ProposalItem.vue:248]: \**

File: src/pages/ProposalItem.vue [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+          name: 'cancel',
+          data: {
+            proposer: proposer.value,
+            proposal_name: proposalName,
+            canceler: account.value
+          }
+        });
+
+        document.location.reload();
+      } catch (e) {
```
Comment (DonaldPeat) [inline src/pages/ProposalItem.vue:275]: \**

File: src/pages/ProposalItem.vue [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+            index: index + 1
+          })
+        );
+
+        const block = await api.getBlock(String(proposal.block_num));
+        transactionId.value = block.transactions[0].trx.id;
+
+        const transaction = await api.getTransaction(transactionId.value);
+        transactionData.value = transaction.actions[0].act.data;
+      } catch (e) {
```
Comment (DonaldPeat) [inline src/pages/ProposalItem.vue:156]: refactor the catch error/notify as its own method and pass the exception as an argument (see ** below)

**Review by DonaldPeat — CHANGES_REQUESTED** [review]

File: src/pages/ProposalItem.vue (hunk 4 above)
Comment (pongilo) [reply src/pages/ProposalItem.vue:170]: Well noticed. Method created!

**Pull request merged**